index.js

```diff
@@ -0,0 +1,15 @@
+const { startOfYesterday, endOfYesterday } = require('date-fns')
```

Comment:
this may or may not work perfect as these are in UTC times when we run through a CI server (like gh actions)